config/defaults.conf

```diff
@@ -413,3 +421,13 @@ edges-constituent-ontonotes = ${edges-tmpl} {
 // These tasks are still very slow. TODO: Debug.
 edges-ccg-tag = ${edges-tmpl}
 edges-ccg-parse = ${edges-tmpl}
+
+// OpenAI transformer versions.
+edges-srl-conll2012-openai = ${edges-srl-conll2012}
```

Comment:
just checking: these should all be defined in `user_config.sh` or wherever people are setting their paths?

Reply:
Sorry, HOCON overloads that syntax - these aren't shell variables, they're just inheriting other tuples. So in this case, it's just setting the `edges-srl-conll2012-openai` config to a copy of the `edges-srl-conll2012` config.